homeassistant/components/ai_speaker/sensor.py

```diff
@@ -0,0 +1,77 @@
+"""Support for AI-Speaker sensors."""
+from datetime import timedelta
+import logging
+
+from aisapi.ws import AisWebService
+
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(minutes=60)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Perform the setup for AI-Speaker status sensor."""
+    _LOGGER.debug("AI-Speaker sensor, async_setup_entry")
+    async_add_entities([AisSensor(hass, config_entry.data)], True)
```

Comment:
Get the web service instance from hass.data here and pass it to the entity when creating that.

Reply:
ok, done in commit @asystentka-jolka
improvements after first review
6214ef8